Source/WebCore/html/parser/HTMLParserIdioms.cpp

```diff
@@ -396,38 +387,29 @@ static bool parseHTTPRefreshInternal(std::span<const CharacterType> data, double
 
         data = data.subspan(1);
 
-        if (!data.empty() && (data.front() == 'R' || data.front() == 'r'))
-            data = data.subspan(1);
-        else {
+        if (!skipExactly(data, 'R') && !skipExactly(data, 'r')) {
             parsedURL = url.toString();
             return true;
         }
 
-        if (!data.empty() && (data.front() == 'L' || data.front() == 'l'))
-            data = data.subspan(1);
-        else {
+        if (!skipExactly(data, 'L') && !skipExactly(data, 'l')) {
             parsedURL = url.toString();
             return true;
         }
 
-        while (!data.empty() && isASCIIWhitespace(data.front()))
-            data = data.subspan(1);
+        skipWhile<isASCIIWhitespace>(data);
 
-        if (!data.empty() && data.front() == '=')
-            data = data.subspan(1);
-        else {
+        if (!skipExactly(data, '=')) {
             parsedURL = url.toString();
             return true;
         }
 
-        while (!data.empty() && isASCIIWhitespace(data.front()))
-            data = data.subspan(1);
+        skipWhile<isASCIIWhitespace>(data);
     }
 
     CharacterType quote;
     if (!data.empty() && (data.front() == '\'' || data.front() == '"')) {
-        quote = data.front();
-        data = data.subspan(1);
+        quote = consumeSingleElement(data);
     } else
```

Comment:
Take off the braces too? I also considered if it would read better using `skipExactly` and it _almost_ does.